core/src/modules/ops/group.ts

```diff
@@ -51,10 +51,11 @@ export namespace GroupOps {
   /**
    * Given a Profile, create an import to recalculate its Group Membership
```

Comment:
I think this comment doesn't quite match up anymore

Reply:
Actually, it still does!  As long as the Profile has `state=pending`, the task to move it to `ready` will also check it's Groups.  I'll clarify the comment.